src/v4/analytics.ts

```diff
@@ -16,6 +16,11 @@ export interface AnalyticsQuery {
   start: Date
 }
 
+export interface AnalyticsRevenueTopProductsQuery extends AnalyticsQuery {
+  orderBy?: string
+  orderDirection?: string
```

Comment:
you don't really need it I guess, but actually there's another query param there:
```
limit?: number
```